bears/hypertext/HTMLLintBear.py

```diff
@@ -27,10 +27,15 @@ class HTMLLintBear:
     CAN_DETECT = {'Syntax', 'Formatting'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file,
-                         htmllint_ignore: typed_list(str)=()):
+    def create_arguments(filename, file, config_file, use_spaces: bool,
+                         htmllint_ignore: typed_list(str)=[]):
         """
         :param htmllint_ignore: List of checkers to ignore.
+        :param use_spaces: True if spaces are to be used instead of tabs.
         """
+        if not use_spaces:
+            htmllint_ignore.append('tabs')
```

Comment:
this is what we shouldn't do: You potentially modify the default value for `htmllint_ignore`. Consecutive calls to `create_arguments` might put the item `tabs` multiple times into the list. You have to create a new list for that^^